packages/plugins/src/foundry/get-bundle-info.ts

```diff
@@ -30,6 +33,19 @@ const userConfig = JSON.parse(userConfigStr)
     const { artifactFolder, buildInfoFolder, canonicalConfigFolder } =
       await getPaths()
 
+    const execAsync = promisify(exec)
+    const forgeConfigOutput = await execAsync('forge config --json')
+    const forgeConfig = JSON.parse(forgeConfigOutput.stdout)
+    const { extra_output }: { extra_output: string[] } = forgeConfig
+    if (
+      !extra_output.includes('storageLayout') ||
+      !extra_output.includes('evm.gasEstimates')
+    ) {
+      throw Error(
+        "foundry.toml file must include both 'storageLayout' and 'evm.gasEstimates' in 'extra_output':\n extra_output = ['storageLayout', 'evm.gasEstimates']"
+      )
+    }
```

Comment:
We check the storage layout and gas estimate options are properly enabled by reading them from the config directly.